node/metrics.go

```diff
@@ -11,22 +12,40 @@ import (
 )
 
 func makeDBMetrics() db.EventListener {
-	readCounter := prometheus.NewCounter(prometheus.CounterOpts{
+	latencyBuckets := []float64{
+		25,
+		50,
+		75,
+		100,
+		250,
+		500,
+		1000, // 1ms
+		2000,
+		3000,
+		4000,
+		5000,
+		10000,
+		50000,
+		math.Inf(0),
+	}
```

Comment:
Shouldn't buckets be either linear or exponential? this is neither, also before going to infinity, a 1-second bucket would be nice, right?

Reply:
> Shouldn't buckets be either linear or exponential?

I don't think that is a requirement. But I don't really know if there is a correct set of values. I set these with by trial and error on my local SSD mainly. We can always update them if it is not making sense for the storage options cloud providers are offering. 

> a 1-second bucket would be nice, right?

adding that as well.